comfort-service.yml

```diff
@@ -182,8 +227,15 @@ namespaces:
             datatype: seat_t
 
         error:
-          datatype: .stdvsc.error_t
-          range: $ in_set("ok", "completed", "in_progress", "interrupted")
+          - description: |
```

Comment:
I think it makes sense to support `description` also for errors, so one can give guidelines on when to use each error message as well as the meaning of it. See also comment below on ok/completed/in_progress.